api/v1alpha1/project_config_types.go

```diff
@@ -20,6 +25,8 @@ type ProjectConfigSpec struct {
 	// PromotionPolicies defines policies governing the promotion of Freight to
 	// specific Stages within the Project.
 	PromotionPolicies []PromotionPolicy `json:"promotionPolicies,omitempty" protobuf:"bytes,1,rep,name=promotionPolicies"`
+	// WebhookReceiverConfigs defines the default receivers for the project.
```

Comment:
Drop "default." You set things up as you like them here and there isn't really any opportunity elsewhere to amend/override, so default feels misleading.